src/SUMMARY.md

```diff
@@ -14,6 +14,13 @@
       - [Using the Data Catalog](cookbooks/analysis/data_catalog.md)
       - [Using the Glean Dictionary](cookbooks/analysis/glean_dictionary.md)
       - [Using the Probe Dictionary](cookbooks/analysis/probe_dictionary.md)
+    - [Data Quality](cookbooks/data_quality/intro.md)
+      - [Bigeye - Data Observability Platform](cookbooks/data_quality/index.md)
```

Comment:
That section is nearly empty and basically the same as the intro section. Remove this line and the file.